src/app/tree-grid-flat-data/tree-grid-flat-data.sample.ts

```diff
@@ -20,7 +20,6 @@ export class MySummaryOperand extends IgxSummaryOperand {
 }
 
 @Component({
-    providers: [{ provide: IgxGridTransaction, useClass: IgxHierarchicalTransactionService }],
```

Comment:
delete also `IgxHierarchicalTransactionService` and `IgxGridTransaction` from the imports.